packages/client/src/pages/profile/avatarModal/ui.tsx

```diff
@@ -1,24 +1,25 @@
-import { Button, Modal, Upload } from 'antd';
 import React, { useState } from 'react';
+import { Button, Modal, Upload, UploadFile } from 'antd';
 import { AvatarModalProps } from './types';
 import { LoadingOutlined, PlusOutlined } from '@ant-design/icons';
+import { setAvatar } from 'pages/profile/model';
+import { UploadChangeParam } from 'antd/es/upload';
 
-export const AvatarModal = ({
+export const AvatarModal: React.FC<AvatarModalProps> = ({
```

Comment:
В проекте должно быть однообразие 
у нас везде просто FC
```suggestion
export const AvatarModal: FC<AvatarModalProps> = ({
```

Reply:
Если единообразия - поправлю. Не вопрос!